recipes/parserutils/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Conservation Biology Institute
```

Comment:
The PyPI tarball already includes this file, so you shouldn't include it in the recipe.